sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Azure.Monitor.OpenTelemetry.Exporter.csproj

```diff
@@ -1,12 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>An OpenTelemetry .NET exporter that exports to Azure Monitor</Description>
     <AssemblyTitle>AzureMonitor OpenTelemetry Exporter</AssemblyTitle>
     <Version>1.1.0-beta.1</Version>
     <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>1.0.0</ApiCompatVersion>
+    <ApiCompatVersion Condition="'$(TargetFramework)' == 'netstandard2.0'">1.0.0</ApiCompatVersion>
```

Comment:
Out of curiosity, why do we need this change?

Reply:
When we shipped last week, the CI added `ApiCompatVersion 1.0.0` to ensure that our Public API doesn't change. (Similar to the PublicApi analyzer we have in other repos).

When I added `net6.0`, the ApiCompat fails because there was no net6.0 in the 1.0.0 release.
I checked with the Azure Experts and they advised to add this condition; we only need to check the api compat for the framework that we shipped already (netstandard2.0).

When we ship our next version, the CI should regenerate this line without the condition. :)